components/rss/sources/common/common.ts

```diff
@@ -0,0 +1,9 @@
+export default {
+  generateMeta: ($, item) => {
+    return {
+      id: $.rss.itemKey(item),
+      summary: item.title,
+      ts: $.rss.itemTs(item),
+    };
+  },
+};
```

Comment:
This is what I would include in `common`.
```suggestion
import rss from "../../app/rss.app";

export default {
  props: {
    rss,
    timer: {
      propDefinition: [
        rss,
        "timer",
      ],
    },
  },
  methods: {
    generateMeta: (item) => {
      return {
        id: this.rss.itemKey(item),
        summary: item.title,
        ts: this.rss.itemTs(item),
      };
    }
  },
};

```